### PR TITLE
Add libgz-utils3-log and dev packages

### PR DESCRIPTION
**Pull request by iche033**
to go with https://github.com/gazebosim/gz-utils/pull/134

### CODE REVIEW

**Review by azeey — COMMENTED** [review]
I think we're implementing it as a component so we'll need to create a `libgz-utils3-log` and `libgz-utils3-log-dev` packages.

**Comment by iche033** [comment]
> I think we're implementing it as a component so we'll need to create a `libgz-utils3-log` and `libgz-utils3-log-dev` packages.

updated to add the these 2 packages.

**Review by caguero — APPROVED** [review]

File: ubuntu/debian/libgz-utils-log.install [hunk 1]
```diff
@@ -0,0 +1 @@
+usr/lib/*/lib*-utils[0-99]-log.so.*
```
Comment (azeey) [inline ubuntu/debian/libgz-utils-log.install:1]: I did a test run and it looks like there are some warnings: https://build.osrfoundation.org/job/gz-utils3-debbuilder/513/console

```
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3 libgz-utils3-log usr/lib/x86_64-linux-gnu/libgz-utils3-log.so.3.0.0
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/include/gz/utils3/gz/utils/log.hh
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/include/gz/utils3/gz/utils/log/Export.hh
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/include/gz/utils3/gz/utils/log/Logger.hh
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/include/gz/utils3/gz/utils/log/SplitSink.hh
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/include/gz/utils3/gz/utils/log/detail/Export.hh
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/lib/x86_64-linux-gnu/cmake/gz-utils3-log/gz-utils3-log-config-version.cmake
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/lib/x86_64-linux-gnu/cmake/gz-utils3-log/gz-utils3-log-config.cmake
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/lib/x86_64-linux-gnu/cmake/gz-utils3-log/gz-utils3-log-targets-relwithdebinfo.cmake
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/lib/x86_64-linux-gnu/cmake/gz-utils3-log/gz-utils3-log-targets.cmake
W: gz-utils3 source: binaries-have-file-conflict libgz-utils3-dev libgz-utils3-log-dev usr/lib/x86_64-linux-gnu/pkgconfig/gz-utils3-log.pc
```

**Review by azeey — CHANGES_REQUESTED** [review]
By the way, I just learned about the possibility of using `curl` to start these builds on jenkins. Here's what I used

```bash
curl -G https://build.osrfoundation.org/job/gz-utils3-debbuilder/buildWithParameters \
  -d 'PACKAGE=gz-utils3&VERSION=nightly&RELEASE_VERSION=1&DISTRO=noble&ARCH=amd64&SOURCE_TARBALL_URI=main&RELEASE_REPO_BRANCH=spdlog_dep&UPLOAD_TO_REPO=none&OSRF_REPOS_TO_USE=stable%20nightly&token=<replace_with_token>'

```

File: ubuntu/debian/libgz-utils-log.install (hunk 1 above)
Comment (iche033) [reply ubuntu/debian/libgz-utils-log.install:1]: ok made a few changes. The build is green now: https://build.osrfoundation.org/job/gz-utils3-debbuilder/517/

**Review by azeey — APPROVED** [review]

**Pull request merged**